sdk/logs/src/test/java/io/opentelemetry/sdk/logs/SdkLogEmitterProviderBuilderTest.java

```diff
@@ -1,42 +0,0 @@
-/*
- * Copyright The OpenTelemetry Authors
- * SPDX-License-Identifier: Apache-2.0
- */
-
-package io.opentelemetry.sdk.logs;
-
-import static io.opentelemetry.sdk.testing.assertj.LogAssertions.assertThat;
-import static org.mockito.Mockito.mock;
-import static org.mockito.Mockito.when;
-
-import io.opentelemetry.sdk.common.Clock;
-import io.opentelemetry.sdk.logs.data.LogData;
-import io.opentelemetry.sdk.resources.Resource;
-import java.util.LinkedList;
-import java.util.List;
-import org.assertj.core.api.AssertionsForClassTypes;
-import org.junit.jupiter.api.Test;
-
-class SdkLogEmitterProviderBuilderTest {
```

Comment:
Moved coverage to `SdkLogEmitterProviderTest` and `SdkLogEmitterTest` for more consistency:
- `SdkLogEmitterProviderTest` contains tests asserting the `SdkLogEmitterBuilder` behavior.
- `SdkLogEmitterTest` contains tests asserting that the configured clock is used to generate the epoch of the log (if the time is not explicitly set).